bin/dswx_landcover_mask.py

```diff
@@ -28,14 +30,14 @@ def _get_parser():
                         help='Input HLS product')
 
     parser.add_argument('-c',
-                        '--copernicus-landcover-100m',
+                        '--landcover', '--land-cover',
                         dest='copernicus_landcover_file',
                         required=True,
                         type=str,
                         help='World cover')
```

Comment:
would you please add a bit more help here to clarify what is this. Specially we want it to be very clear and distinguished from the next one (--world-cover)